### DISCUSSION TITLE
Metrik für eine "Wir haben Fertig" Behauptung für die TdSE Freigabe auf GitHub?

### DISCUSSION THREAD

**Discussion by clalitsc** [GfSE/SAF-Specification#38] [Q&A] [answered]
Welche Metrik(en) für eine "Wir haben Fertig" Behauptung für die anstehende SAF TdSE 2023 Freigabe auf GitHub ziehen wir an?
- Das Vorhandensein der geforderten SAF VPs für ein PDR, siehe SAF Workshop Februar 2023, vgl.
https://github.com/GfSE/SAF-Specification/blob/main/using-saf/preliminarydesignreview.md#viewpoints-new-for-pdr
- Die Umsetzung der SAF Unterstützung in den kommerziellen SysML Autoren-Werkzeugen, vgl.
https://github.com/GfSE/SAF#a-saf-profilspezifikation-f%C3%BCr-sysml
- Ein erfolgreiches Review der 37 VP Spezifikationstexte und Ontologie Abbildungen? SCV=1, SOV=12, SFV=13, SLV=4, SPV=7 +/-
- Lasst uns sehen, wie weit wir bis zum 15.11. EoB kommen? Agil?
- ...
@GfSE/saf

**Comment by haarer** [comment, marked as answer]
Meine Meinung:

Die VP zum PDR sind ja - bis auf zwei, die dort nicht verlinkt sind drin. Die beiden fehlenden
"Phyiscal Logical Interface Assignment Viewpoint" und "Subsystem Requirement Specification Viewpoint" werden wir nicht fertig stellen können.

Da wir die Konzeptmodelle als Teil der Developer Doku hier im Repository haben und die auch jedesmal wenn die Doku aktualisiert wird auf Stand ist, würde ich nicht erwarten daß es hier ein extra review braucht.
Jeder kann die laufende Entwicklung mitlesen, und sich ggfs melden. Wir haben die Rückmeldungen die wir bekommen haben so weit möglich eingearbeitet.

Ich hatte mir das eigentlich so vorgestellt, daß die Developer Doku (Ontologie) nicht Teil des releases sein muss. Das Verzeichnis würde ich einfach aus dem release Branch rauswerfen. Entwickelt werden soll ja eh nur im main Branch.

Für die Profilentwicklung ist m.E. entscheidend, daß sich die Stereotypen nicht mehr ändern.
Das können wir m.E. zusichern. Das einzige was sich dort ändern mag ist redaktionelles (also die Dokumentation der Stereotypen).

Ich denke wir sollten in den nächsten Tagen den branch anlegen, auch in den Profilrepositories.
Wir sollten aber vorher den aktuellen Stand der profile in main der jeweiligen repositories ablegen. Im Moment liegt dort ja der Stand vom letzten jahr.

Für die Beispielmodelle hatten wir ja schon besprochen, daß die nicht Teil des Releases werden.

**Reply by haarer** [reply]
Ich habe jetzt für die Spec den TdSE2023 branch angelegt.
Möglicherweise ist es ja eine gute Idee im Readme noch was zu schreiben ? 
https://github.com/GfSE/SAF-Specification/tree/TdSE2023

Auf der folgenden Seite benennen wir die Branches (auch wenn wir sie nicht einzeln verlinken).
Dort sollte auch noch der Branch benannt werden. 
https://github.com/GfSE/SAF
@GfSE/saf

**Reply by clalitsc** [reply]
Ein noch bessere Idee wäre es das Readme kritisch gegenzulesen und es fachlich wie inhaltlich zu verbessern. 
Abschnitt: [Wer trägt SAF ?](https://github.com/GfSE/SAF#wer-tr%C3%A4gt-saf-)
"Es wird von den Mitgliedern [...]", hier sollten die beteiligten Firmen analog zu TdSE 2023 Plakat genannt werden, mit oder ohne Logo.

**Reply by haarer** [reply]
man könnte ja das plakat oder einen teil davon dort ikonisch einbauen ?
